server/controllers/items/by_user_and_entity.coffee

```diff
@@ -28,7 +28,7 @@ getUser = (query, reqUserId)->
     return user_.getUserById userId, reqUserId
 
   else if username?
-    unless tests.username username
+    unless isValid username
```

Comment:
this should crash: `Username` and thus `isValid` is a regex, not a function.
Being function is the role of `__.require('models', 'validations/user').username`, so I would prefer something like this:
```coffee
{ username: isValidUsername } = __.require 'models', 'validations/user'
[...]
unless isValidUsername username
```